plugin/evm/config.go

```diff
@@ -191,6 +191,11 @@ type Config struct {
 	// Database Settings
 	InspectDatabase bool `json:"inspect-database"` // Inspects the database on startup if enabled.
 
+	// OutOfBandWarpMessagesBytes represent off-chain warp messages (in bytes) a validator is willing to sign
+	// These messages are not tied to on-chain events and are particularly useful for implementing a unified
+	// upgrade mechanism for critical smart contracts on the chain.
+	OutOfBandWarpMessagesBytes [][]byte `json:"out-of-band-warp-messages"`
```

Comment:
nit: 
hmm this feels like:
![image](https://github.com/ava-labs/subnet-evm/assets/18367708/b100eec2-8c64-4fa5-964b-78968193e557)
Can we name it `OffChainWarpMessagesBytes`?

Reply:
sure